content/guide/collection-catalouges.md

```diff
@@ -86,7 +92,7 @@ In a collection catalogue, there will typically be a visual table of contents fo
   📄 3.md
 ```
 
-The `overview.md` file is going to be our visual table of contents. To populate it, simply give it a `type` of "contents" and a `class` of  "brief", "list", "abstract", or "grid" to determine the style. (The "grid" option will include an image from each entry page.) This "contents" page will automatically generate from each of the Markdown files in the folder.
+The `overview.md` file is going to be our visual table of contents. To populate it, simply give it the attribute `type` a value of `contents` and the attribute `class` a value of  `brief`, `list`, `abstract`, or `grid` to determine the style. (The `grid` option will include an image from each entry page.) This `contents` page type will automatically generate from each of the Markdown files in the folder.
```

Comment:
This is the first time I noticed this, but should we differentiate the grammatical/stylistic way we indicate a yaml **key** versus as yaml **value**? So, in this case, the key is `class` and the value is `brief`, `list`, etc. When we have a value should we keep the straight double quotes and set it in monospace? So, `"brief"`, `"list"`, etc. Does that help distinguish the two types of information (what is a key and what is a value) and is having them distinguished helpful? Or is it unnecessary and messy? I'm not sure. Are there recommendations on this from some of the documentation sources you were referring to?

Reply:
Now that you point it out, I agree about the differentiation between keys and values. 

The Google developers documentation style guide suggests the use of code font and double quotation marks enclosing for XML attribute values: https://developers.google.com/style/api-reference-comments 

I can keep the quotation marks and add the code font syntax.

Reply:
Perfect. Can you also update the styling to match in the Contributors and the Bibliographies pull requests you have open? Ping me when they're all updated and I'll look at merging those as well. Thanks!